lib/airtable.ts

```diff
@@ -0,0 +1,145 @@
+
+import Airtable from 'airtable';
+require('dotenv').config();
+
+const base = new Airtable({ apiKey: process.env.AIRTABLE_API_KEY }).base('appUW06bs08YxzVDM')
+
+const getIdea = async ({ id }) => {
+  try {
+    // 1. Find Idea by by ID   
+    const record = await base('Ideas').find(id)
```

Comment:
miss `;` at this line? @kelvin2go

Reply:
es6 allow no ; 
normally  set a standard all no or all have. 

lets set all have